README.md

```diff
@@ -23,13 +23,13 @@ The current version of pin-project requires Rust 1.33 or later.
 
 ## Examples
 
-[`unsafe_project`] attribute creates a projection struct covering all the fields.
+[`pin_projectable`] attribute creates a projection struct covering all the fields.
 
 ```rust
-use pin_project::unsafe_project;
+use pin_project::pin_projectable;
 use std::pin::Pin;
 
-#[unsafe_project(Unpin)] // `(Unpin)` is optional (create the appropriate conditional Unpin implementation)
+#[pin_projectable(Unpin)] // `(Unpin)` is optional (create the appropriate conditional Unpin implementation)
```

Comment:
What does it do without the `(Unpin)`?

Reply:
Oops - I meant to remove that. The README doesn't get tested when `cargo doc` is run, so I never saw an error.

Reply:
See https://github.com/abonander/readme-doctest-poc for how to test the README using rustdoc :D

Reply:
Or https://crates.io/crates/doc-comment for a stable version.

Reply:
Filed #19. Thanks for the info @RalfJung and @shepmaster!